README.md

```diff
@@ -39,7 +39,7 @@ to capture detailed changes at the individual row level.
 
 | name     | description                                                                                                                                         | required | default value |
 | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | ------------- |
-| `url`    | The connection URL of the MySQL, in the following format (\*): [username[:password]@][protocol[(address)]]/dbname[?param1=value1&...&paramN=valueN] | true     |               |
+| `url`    | The connection URL of the MySQL, in the following format: `[username[:password]@][protocol[(address)]]/dbname[?param1=value1&...&paramN=valueN]`    | true     |               |
```

Comment:
This points to the `(*)` below the table. We could remove it, but then I suggest including the link somewhere here directly, maybe like this.

```suggestion
| `url`    | The connection URL of the MySQL, in the [following format](https://github.com/go-sql-driver/mysql?tab=readme-ov-file#dsn-data-source-name): `[username[:password]@][protocol[(address)]]/dbname[?param1=value1&...&paramN=valueN]`    | true     |               |
```